src/components/Testing.tsx

```diff
@@ -16,27 +48,66 @@ export default function Testing(user : Struct.User) {
   // function test (){
   //     console.log('this works yay!!');
   //     // number = (number + 10 ) %20;
+
+
   function LogEvent(){
```

Comment:
What is this for? is this linked to supabase dummy event? it is showing import error from my side

Reply:
nvm, that's nothing important, just some dummy functions I did to conduct some testing, which will be removed before the project is complete. The testing.tsx file in mainly for my testing purposes.